ooniapi/services/ooniprobe/src/ooniprobe/routers/v1/probe_services.py

```diff
@@ -590,3 +589,46 @@ def random_web_test_helpers(th_list: List[str]) -> List[Dict]:
     for th_addr in th_list:
         out.append({"address": th_addr, "type": "https"})
     return out
+
+
+class GeoLookupResponse(BaseModel):
+    """
+    v:
+      type: integer
+      description: response format version
+    geolocation:
+      type: object
+      description: dict of ip addresses to dict of country code, ASN, AS Name strings
+    """
+    v: int
+    geolocation: Dict[str, Dict[str, str]]
```

Comment:
Since the structure of `geolocation` is fixed you should use a model for the values of this dict: 

```python
geolocation: Dict[str, Geolocation] # for example, feel free to chose a different name 

...

class Geolocation(BaseModel):
 ...
```